grpc/client/src/test/java/io/helidon/grpc/client/GrpcChannelsProviderTest.java

```diff
@@ -0,0 +1,197 @@
+    private static final String DEFAULT_HOST_SSL_ONE_WAY_CFG = "default_host_ssl_one_way";
+    private static final String DEFAULT_PORT_SSL_CFG = "default_port_ssl";
+    private static final String DEFAULT_HOST = "localhost";
+    private static final int DEFAULT_PORT = 1408;
+    private static GrpcChannelsProvider grpcConfig;
+
+    @BeforeAll
+    public static void initGrpcConfig() {
+        Config cfg = Config.create(ConfigSources.classpath("test-client-config.yaml"));
+        grpcConfig = GrpcChannelsProvider.create(cfg.get("grpc"));
+    }
+
+    @Test
+    public void testDefaultChannelConfiguration() {
+        GrpcChannelDescriptor cfg = GrpcChannelDescriptor.builder().build();
+        assertThat(cfg.host(), equalTo(DEFAULT_HOST));
+        assertThat(cfg.port(), equalTo(DEFAULT_PORT));
+        assertThat(cfg.sslDescriptor(), nullValue());
+    }
+
+    @Test
+    public void testChannelConfigurationWithHost() {
+        GrpcChannelDescriptor cfg = GrpcChannelDescriptor.builder().host("abc.com").build();
+        assertThat(cfg.host(), equalTo("abc.com"));
+        assertThat(cfg.port(), equalTo(DEFAULT_PORT));
+        assertThat(cfg.sslDescriptor(), nullValue());
+    }
+
+    @Test
+    public void testChannelConfigurationWithPort() {
+        GrpcChannelDescriptor cfg = GrpcChannelDescriptor.builder().port(4096).build();
+        assertThat(cfg.host(), equalTo("localhost"));
+        assertThat(cfg.port(), equalTo(4096));
+        assertThat(cfg.sslDescriptor(), nullValue());
+    }
+
+    @Test
+    public void testChannelConfigurationWithDefaultSsl() {
+        GrpcChannelDescriptor cfg = GrpcChannelDescriptor.builder()
+                .sslDescriptor(GrpcSslDescriptor.builder().build())
+                .build();
+        assertThat(cfg.host(), equalTo("localhost"));
+        assertThat(cfg.port(), equalTo(1408));
+        assertThat(cfg.sslDescriptor().isEnabled(), is(true));
+    }
+
+    @Test
+    public void testChannelConfigurationWithSslConfig() {
+        GrpcChannelDescriptor cfg = GrpcChannelDescriptor.builder()
+                .sslDescriptor(
+                        GrpcSslDescriptor.builder()
+                                .tlsCaCert("/certs/cacert")
+                                .tlsCert("/certs/clientcert")
+                                .tlsKey("/certs/clientkey")
+                                .build())
+                .build();
+        assertThat(cfg.host(), equalTo("localhost"));
+        assertThat(cfg.port(), equalTo(1408));
+        assertThat(cfg.sslDescriptor().isEnabled(), is(true));
+        assertThat(cfg.sslDescriptor().tlsCaCert(), equalTo("/certs/cacert"));
+        assertThat(cfg.sslDescriptor().tlsCert(), equalTo("/certs/clientcert"));
+        assertThat(cfg.sslDescriptor().tlsKey(), equalTo("/certs/clientkey"));
+    }
+
+    // "default_host_port", "default_host", "default_port", "default_host_port_ssl_disabled", "default_host_ssl_one_way",
```

Comment:
Is this left-over or is it actually intended?

Reply:
I've cleaned this up.